net-libs/nodejs/nodejs-16.13.1.ebuild

```diff
@@ -49,6 +49,7 @@ DEPEND="${RDEPEND}"
 PATCHES=(
 	"${FILESDIR}"/${PN}-12.22.5-shared_c-ares_nameser_h.patch
 	"${FILESDIR}"/${PN}-15.2.0-global-npm-config.patch
+	"${FILESDIR}"/${PN}-16.13.1-riscv-arch.patch
```

Comment:
The last commit is adding back this patch which was dropped 2 commits before that, can you squash this change please (i.e. don't drop the patch at all)

Reply:
Thx for reminding. Done.